lib/organization.js

```diff
@@ -113,7 +121,9 @@ export class Organization {
       initialSchemesParams.push(paramsHash);
       initialSchemesTokenAddresses.push(await scheme.nativeToken());
       initialSchemesFees.push(await scheme.fee());
-      initialSchemesPermissions.push(scheme.getDefaultPermissions(/* supply options.permissions here? */));
+      initialSchemesPermissions.push(
+        scheme.getDefaultPermissions(/* supply options.permissions here? */)
```

Comment:
why default ? It should be given as param to the function

Reply:
I think there is minimum permission per scheme according to what it is doing.
e.g a SchemeRegistrar scheme should at least has registrar permission.(0x3)..and so on..
But if a user would like to create an org with a "special" scheme it should be able to give it a permission it needed.